### DISCUSSION TITLE
UI issue on sign-up screen

### DISCUSSION THREAD

**Discussion by umer2001** [community/community#19106] [Copilot Conversations] [answered]
There is a CSS issue with text color on the signup page (https://github.com/github-copilot/tp_signup)
text having issue:
GitHub Copilot
Your AI pair programmer

image
![copilot](https://user-images.githubusercontent.com/35496058/174910301-08570eb5-3fcf-492d-9453-649ef57ad176.PNG)

_________________
I am a student, If possible provide me GitHub copilot for free for reporting this issue.

**Comment by melvincwng** [comment, marked as answer]
Hi @umer2001, you can report bugs to GitHub Bug Report over [here](https://support.github.com/contact/bug-report)

Hope this clarifies.

**Reply by melvincwng** [reply]
@umer2001, if you find this reply useful, please kindly 'Mark as answer' so the rest are aware this issue has been addressed.

Thank you and have a nice day 🙏🏻
